### PR TITLE
Add Image Label Buildpack

### LINKED ISSUE #5: Add Image Labels Buildpack to Group Order
We should add the [Image Labels Paketo Buildpack](https://github.com/paketo-buildpacks/image-labels) to the group order so that the PHP Paketo Buildpack conforms to the [image labels RFC](https://github.com/buildpacks/rfcs/blob/master/text/0034-image-labels.md)

### PR DESCRIPTION
**Pull request by sophiewigmore**
## Summary

Resolves #5. This is based off the [env-var-bp](https://github.com/paketo-buildpacks/php/tree/env-var-bp) branch for PR #328, if that can get merged in first. Alternatively I can close that PR and merge these two together. 

## Use Cases

Adds image label buildpack to order grouping!

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have added an integration test, if necessary.

### CODE REVIEW

**Comment by arjun024** [comment]
@sophiewigmore Could you rebase on the current main please?

**Comment by sophiewigmore** [comment]
@arjun024 Done

File: buildpack.toml [hunk 1]
```diff
@@ -42,3 +42,8 @@ api = "0.2"
     id = "paketo-buildpacks/environment-variables"
     optional = true
     version = "3.0.0"
+
+  [[order.group]]
+    id = "paketo-buildpacks/image-labels"
+    optional = true
+    version = "3.0.0"
```
Comment (arjun024) [inline buildpack.toml:49]: Could you please add a newline to terminal the file?

File: package.toml [hunk 2]
```diff
@@ -22,3 +22,6 @@
 
 [[dependencies]]
   uri = "docker://gcr.io/paketo-buildpacks/environment-variables:3.0.0"
+
+[[dependencies]]
+  uri = "docker://gcr.io/paketo-buildpacks/image-labels:3.0.0"
```
Comment (arjun024) [inline package.toml:27]: Could you please add a newline to terminal the file?

**Review by arjun024 — CHANGES_REQUESTED** [review]

**Review by arjun024 — APPROVED** [review]

**Pull request merged**